### ISSUE TITLE
Upload multiple files?

### ISSUE THREAD

**Issue by xzhub** [heryandi/flask-s3-multipart#1]
Hi heryandi,
This project is very useful, thanks for sharing! however, I did some tests but failed to implement the multiple file uploading based on this project(I'm still learning JS). It would be very appreciated if you could modify your code to support multiple files uploading. I am sure many others will also benefit from your code. Thanks in advance! :)

**Comment by nav347** [comment]
Hey  @xzhub,
It seems you got this project working for concurrent uploads in 2019. Do you think it would be possible to help me out with getting it to work? Right now receiving Error 400

**Comment by xzhub** [comment]
I eventually used this project, I believe you will like it too: https://fineuploader.com/

**Comment by nav347** [comment]
Yes,fineuploader for python3 flask works fine for single uploads but gives signature does not match error when chunking. Were you able to resolve it?

**Comment by xzhub** [comment]
I don't remember I encountered that issue. Could you please raise a bug report in their github page? I am sure someone will help.

**Comment by nav347** [comment]
Ahh, unfortunately the project has stopped receiving support and I am not able to figure out yet the error. Gonna convert this to flask backend and try to use this instead
